debian/control

```diff
@@ -8,5 +8,5 @@ Standards-Version: 4.1.2
 Package: wolkgateway
 Architecture: any
 Section: main
-Depends: ${shlibs:Depends}, ${misc:Depends}, libssl1.0.0 | libssl1.1 | libpoco-dev
+Depends: ${shlibs:Depends}, ${misc:Depends}, libssl1.0.0 | libssl1.1 | libssl-dev
```

Comment:
libssl-dev already implies libssl1.0.0 or libssl1.1 and we shouldn't depend on dev library when running